apps/avatax/src/logger-context.ts

```diff
@@ -3,4 +3,4 @@ import { LoggerContext } from "@saleor/apps-logger/node";
 /**
  * Server-side only
  */
-export const loggerContext = new LoggerContext();
+export const loggerContext = new LoggerContext("saleor-app-avatax");
```

Comment:
we should pass this name from env etc. Maybe there is Vercel env with project name? If not, create one

Reply:
So maybe we should pass here `OTEL_SERVICE_NAME` too? 🤔

Reply:
For consistency I think so